src/code/z_lights.c

```diff
@@ -317,42 +317,36 @@ Lights* Lights_New(GraphicsContext* gfxCtx, u8 ambientR, u8 ambientG, u8 ambient
 }
 
 void Lights_GlowCheck(PlayState* play) {
-    LightNode* node;
-    LightPoint* params;
-    Vec3f pos;
-    Vec3f multDest;
-    f32 cappedInvWDest;
-    f32 wX;
-    f32 wY;
-    s32 wZ;
-    s32 zBuf;
+    LightNode* light = play->lightCtx.listHead;
 
-    node = play->lightCtx.listHead;
+    while (light != NULL) {
+        LightPoint* params = &light->info->params.point;
 
-    while (node != NULL) {
-        params = &node->info->params.point;
+        if (light->info->type == LIGHT_POINT_GLOW) {
+            Vec3f pos;
+            Vec3f multDest;
+            f32 wDest;
 
-        if (node->info->type == LIGHT_POINT_GLOW) {
             pos.x = params->x;
             pos.y = params->y;
             pos.z = params->z;
-            Actor_ProjectPos(play, &pos, &multDest, &cappedInvWDest);
+            Actor_ProjectPos(play, &pos, &multDest, &wDest);
```

Comment:
`Actor_ProjectPos`'s last argument in the Inverse of `w`, not `w` afaik. So the `Inv` shouldn't be dropped. And it's capped too, so I think the original name was fine:
https://github.com/zeldaret/oot/blob/9647f50451f26fa8b96653bf792f86c6148f6d12/src/code/z_actor.c#L219